contracts/contract_state_tree.md

```diff
@@ -13,20 +14,51 @@ The contract state tree contains objects:
 - The contract log (data binary)
 - Active flag (Boolean)
 - List of refering contracts
-- Calls & Return data
 - Deposit (amount)
 - vm_version
 
+It is serialized as:
+```
+      CONTRACT_TYPE,
+      CONTRACT_VSN,
+      pubkey,
+      balance,
+      height,
+      nonce,
+      owner,
+      vm_version,
+      code,
+      state,
+      log,
+      active,
+      referers,
+      deposit,
+```
+
 As well as normal account fields.
 
 #### The contract definition
 
 - Created by a contract create transaction.
 - Deleted when an inactive contract has no more referers.
 
-
 ### Contract state tree update
 
+### Pruning of contracts
 
+## Calls state tree
 
-### Pruning of contracts
+The calls state tree contains all calls done in the block
+and their return values.
+
+- Calls & Return data
+```
+      	CONTRACT_INTERACTION_TYPE,
+      	CONTRACT_INTERACTION_VSN,
+      	caller_address,
```

Comment:
I suggest deleting this paragraph and reviewing https://github.com/aeternity/protocol/blob/b9b9eb2a707a18b7b976be78f3eca76e27c9c5e3/serializations.md#contract-call - there are two nonces there.